packages/web-components/fast-components/src/styles/patterns/button.ts

```diff
@@ -187,7 +171,7 @@ export const AccentButtonStyles = css`
         color: ${accentForegroundCutRestBehavior.var};
     }
 
-    :host([appearance="accent"]:hover) {
+    :host([appearance="accent"]:hover:) {
```

Comment:
Typo?

Reply:
thanks. a PR will up shortly to fix this